notebooks/106-auto-device/106-auto-device.ipynb

```diff
@@ -0,0 +1,407 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "190e8e4c-461f-4521-ae7f-3491fa827ab7",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Automatic Device Selection\n",
+    "\n",
+    "The Auto Device (or AUTO in short) selects the most suitable device from the available compute devices by considering the network precision, power efficiency and processing capability. The network precision (if the network is quantized or not) is the first consideration to filter out the devices that cannot run the network efficiently.\n",
+    "\n",
+    "Next, the dedicated accelerator devices are preferred, e.g., discrete GPU, integrated GPU, or VPU. CPU is used as the default “fallback device”. Please note that AUTO does this selection only once at the network load time. \n",
+    "\n",
+    "When choosing the accelerator device like GPUs, loading the network to these devices may take long time. To address this challenge for application that requires fast initial inference response the AUTO starts inferencing immediately on the CPU and then transparently shifts inferencing to the GPU once ready, dramatically reducing time to first inference.\n",
+    "\n",
+    "![Auto Device Selection logic](data/auto_device_selection.png \"Auto Device Selection\")\n",
+    "\n",
+    "The following demostrations use the [googlenet-v1](https://docs.openvino.ai/latest/omz_models_model_googlenet_v1.html) model from the [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/). The googlenet-v1 model is the first of the Inception family of models designed to perform image classification. Like the other Inception models, the googlenet-v1 model has been pre-trained on the ImageNet image database. For details about this family of models, check out the paper.\n",
+    "\n",
+    "Please follow the [104-model-tools](../104-model-tools/README.md) to download googlenet-v1 and convert (--precisions FP16), copy the results (googlenet-v1.bin and googlenet-v1.xml) to notebooks\\106-auto-device\\model\\ folder"
```

Comment:
Instead of this instructions, it's better to provide code cells with downloading and converting the model (see e.g. 401)

Reply:
Please accept the @adrianboguszewski suggestion. Otherwise, the first impression will be this one.

![image](https://user-images.githubusercontent.com/10940214/160919085-2e74e47a-3342-4aea-9700-6488da72f501.png)

Reply:
Added the downloading and converting code.